### ISSUE TITLE
KeyError: 'Event'

### ISSUE THREAD

**Issue by jaegeral** [DCSO/tie2misp#18]
Querry URL: https://tie.dcso.de/api/v1/iocs?category=c2-server&created_since=2017-06-23&created_until=2017-06-24
Parsing... - Index: 0
Attribute: 10 from 35
Attribute: 20 from 35
Attribute: 30 from 35
Attribute: 35 from 35

````
 File "./tie2misp", line 79, in <module>
    startup()
  File "/usr/local/lib/python3.4/dist-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.4/dist-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.4/dist-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.4/dist-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "./tie2misp", line 72, in startup
    Loader.start(conf, tags, type, dt, file, noupload)
  File "/usr/local/bin/tie2misp/loader.py", line 108, in start
    event.upload(conf)
  File "/usr/local/bin/tie2misp/model/misp_event.py", line 153, in upload
    attr.upload(misp, event, config)
  File "/usr/local/bin/tie2misp/model/attributes/ip_v4.py", line 46, in upload
    self.upload_tags(misp, attr)
  File "/usr/local/bin/tie2misp/model/misp_attribute.py", line 136, in upload_tags
    attr_list = event['Event']['Attribute']
KeyError: 'Event'

````

**Comment by Mezzonian** [comment]
Working on it. Should be fixed next week within the update for the confidence and severity filter.

**Comment by Mezzonian** [comment]
There were some changes in the new version of PyMISP that lead to this error. In my development environment, I used a slightly older version in which the error has not yet occurred in my tests. I've adjusted this in my local branch now, so this should be fixed with the next merge on Monday.

**Comment by Mezzonian** [comment]
Problem should be fixed with the merge #19
